config/feed.json

```diff
@@ -12,9 +12,7 @@
     "setzerTimeout": 10,
     "setzerCacheExpiry": 120,
     "setzerMinMedian": 3,
-    "setzerEthRpcUrl": "https://mainnet.infura.io/v3/fac98e56ea7e49608825dfc726fab703",
-    "verbose": true,
-    "logFormat": "text"
+    "setzerEthRpcUrl": "https://mainnet.infura.io/v3/fac98e56ea7e49608825dfc726fab703"
```

Comment:
any concern about storing api keys clear in the repo?

Reply:
It's been here for a while, so not this time :)

Reply:
heh, yeah.  at my previous engagement we made use of https://github.com/mozilla/sops to store secrets in the repo, only having to share a master key amongst devs. pretty great tool since you could integrate it into Makefiles, env vars, substitutions, etc.

Reply:
There is one place we use git-crypt